docs/scripting-api.rst

```diff
@@ -874,13 +882,21 @@ omitted.
          sleep(500)
        }
 
-.. js:function:: iconTag([tag])
+.. js:function:: String iconTag()
+
+   Get current tray and window tag text.
+
+.. js:function:: iconTag(tag)
+
+   Set current tray and window tag text.
+
+.. js:function:: String iconTagColor()
 
-   Get or set current tray and window tag text.
+   Get current tray and window tag color name.
 
-.. js:function:: iconTagColor([colorName])
+.. js:function:: iconTagColor()
```

Comment:
Missing `colorName` argument.

Reply:
Argh, I did the same error on iconColor(), and fixed it in time, but this one went unnoticed. Thanks.